docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -199,3 +199,19 @@ In addition to the above methods, broadly used the next approaches and common is
 include::troubleshooting/alerting-common-issues.asciidoc[]
 include::troubleshooting/event-log-index.asciidoc[]
 include::troubleshooting/testing-connectors.asciidoc[]
+
+
+[float]
+[[alerting-kibana-disable]]
+=== Temporarily disable
+
+A cluster may become unresponsive or sluggish if too many or expensive {kib} rules 
+are attempting to run. As a stop gap measure, you may consider temporarily overriding
+the {kib} Task Manager to gain breathing room to resolve your situation by restarting 
+{kib} after updating its config to
+
+[source, txt]
+--------------------------------------------------
+xpack.task_manager.max_workers: 1
+xpack.task_manager.poll_interval: 1m
+--------------------------------------------------
```

Comment:
I think we should extend with two things:

- what this is doing
- remember to reset it after resolving the problem!

# what this is doing

Setting `xpack.task_manager.max_workers: 1` will limit each {kib} instance to claiming one task per polling cycle.  The default is 10.

Setting `xpack.task_manager.poll_interval: 1m` sets the {kib} Task Manager polling cycle to 1 minute.  The default is 3 seconds (`3s`)

Reply:
This seems like a duplicate of what we already have in https://www.elastic.co/guide/en/kibana/master/task-manager-settings-kb.html

Perhaps instead we send them to those docs for more details, rather than explaining them here again?

Reply:
I've added a link to the settings page in e9fc3a5a40e234b99611f15e2fbc08282d8ba0ae. However, I agree that it's not clear from a glance what these recommended changes accomplish. And I think we need to reiterate that you should revert these changes when your troubleshooting is complete.

Reply:
Thanks Lisa! 👍